core/domain/question_domain.py

```diff
@@ -147,8 +147,41 @@ def validate(self):
             raise utils.ValidationError(
                 'Expected question_data to be a dict, received %s' %
                 self.question_data)
+
+        #return self.question_data['interaction']
```

Comment:
Do we need this?

Reply:
question_data is a State object, so it has to be a dict , that's why I put the check, just as a precaution. Should it be removed?

Reply:
I think @DubeySandeep was specifically asking whether we need this comment. Since it doesn't seem to be providing any additional context, can we just remove it?

Reply:
I think @DubeySandeep was asking about the commented-out line 151.

Reply:
Oh, ok, will remove that.

Reply:
Done